### PR TITLE
Add view sched v1.2

### PR DESCRIPTION
**Pull request by Gabau**
Current status: WIP, scrapped previous implementation, currently text based implementation

- [x] Create placeholder class
- [x] **Implement getSchedule in person class**
- [x] Setup output for scheduler (Currently text output)

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ViewScheduleCommand.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    private final PersonContainsFieldsPredicate predicate;
+
+    public ViewScheduleCommand(PersonContainsFieldsPredicate predicate) {
+        this.predicate = predicate;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireAllNonNull(model);
```
Comment (IrvingHe000) [inline src/main/java/seedu/address/logic/commands/ViewScheduleCommand.java:33]: should it be requireNonNull?

File: src/main/java/seedu/address/model/ModelManager.java [hunk 2]
```diff
@@ -129,6 +129,12 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         filteredPersons.setPredicate(predicate);
     }
 
+    @Override
+    public void updateScheduleDisplay(Predicate<Person> predicate) {
```
Comment (fullfatwasabi) [inline src/main/java/seedu/address/model/ModelManager.java:133]: Is this meant to be implemented?

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -27,8 +27,8 @@
         <URL value="@Extensions.css" />
       </stylesheets>
 
-      <VBox fx:id="mainVbox" prefHeight="471.0" prefWidth="925.0">
-        <MenuBar fx:id="menuBar" prefHeight="75.0" prefWidth="932.0" VBox.vgrow="NEVER">
```
Comment (fullfatwasabi) [inline src/main/resources/view/MainWindow.fxml:31]: Hmm is this relevant to this pr

**Review by fullfatwasabi — CHANGES_REQUESTED** [review]
LGTM except two parts heh

**Review by IrvingHe000 — APPROVED** [review]
LGTM!

**Pull request merged**